docs/yymnist/README.md

```diff
@@ -0,0 +1,80 @@
+README
+==========
+
+# Introduction
+
+This document is intended to walk a user through the necessary steps to run object detection on the [yymnsit](https://github.com/cmu-sei/yymnist) dataset with Juneberry.
+
+# Setup 
+Install opencv, Juneberry, and pull down the yymnsit repo
+<br>
+`git config --global http.proxy http://cloudproxy.sei.cmu.edu:80/`
+`
+git clone https://github.com/cmu-sei/yymnist.git
+`
+# Create your datasets
+The yymnist repository has a script that allows a user to create reproducible datasets to test.
+
+1. We first need to generate our train/test/val datasets. 
+
+For this example we will do a 80/10/10 split respectively.
+
+Training Set:
```

Comment:
Can we make a script with all six commands?

Reply:
Script now generates all three in a single call